README.md

```diff
@@ -1,32 +1,77 @@
-## Getting Started
+<div align="center">
 
-### Init odra module.
+<h1>Odra</h1>
+<h3>High-level smart contract framework for Rust.</h3>
 
-```bash
-cargo odra new mycontract
-```
+</div>
 
-### Build wasm
-```bash
-cargo build --release --no-default-features --features "wasm"
-cargo odra build-wasm -p mycontract
-```
+## Table of Contents
+* [Usage](#usage)
+* [Example](#example)
+* [Backends](#backends)
+* [Links](#links)
+* [Contact](#contact)
 
-### Run tests against MockVM
+## Usage
+TODO: Add gif with example.
 
-```bash
-cargo test -p mycontract
-```
+TODO: Show casper-odra commands.
+
+## Example
+
+```rust
+use odra::Variable;
+
+#[odra::module]
+pub struct Flipper {
+    value: Variable<bool>,
+}
 
-### Run tests against CasperVM
-```bash
-cargo odra test-with-backend -p mycontract
+#[odra::module]
+impl Flipper {
+    pub fn flip(&self) {
+        self.value.set(!self.get());
+    }
+
+    pub fn get(&self) -> bool {
+        self.value.get_or_default()
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::Flipper;
+
+    #[test]
+    fn flipping() {
+        let contract = Flipper::deploy();
+        assert_eq!(contract.get(), false);
+        contract.flip();
+        assert_eq!(contract.get(), true);
+    }
+}
 ```
 
-## Modes
+## Backends
+
+Odra is designed to integrate any most WebAssemby-based smart contract systems.
```

Comment:
Any or most?

Reply:
Most, because if smart contract system is not account-based like Bitcoin, it has totally different concepts.